guides/plugins/plugins/content/cms/real-world-cms/add-real-world-cms-element.md

```diff
@@ -0,0 +1,483 @@
+# Add a real world CMS element
+
+## Overview
+
+This guide is based on the [Add a real world CMS block](add-real-world-cms-block.md) guide. We will create a real world CMS element in this guide.
+That means we are creating an element with a config which fits to our CMS-Button block.
+
+## Prerequisites
+
+We assume that you know the basics for about creating a CMS element from [Creating a custom CMS element](../add-cms-element.md). So if you don't understand
+something please make sure to take a look at that guide as well.
+
+## Creating your custom CMS element
+
+Imagine you want to create a new element to display a button which is fully configurable by the shop manager. We already build a CMS-Block for the button in the
+last guide but if you just have a block you don't have a configuration that's why you need a CMS element
+
+## Administration Code
+
+Since you already know the basics from [Add CMS Element](../add-cms-element.md), we will just look a quick look at the code.
+If you followed the real world block guide you already should have a `main.js` in the `<plugin root>/src/Resources/app/administration/src` directory.
+In this file we need to import our new element's directory. So let's do it:
```

Comment:
```suggestion
We will look at the code briefly since you already know the basics from [Add CMS Element](../add-cms-element.md).
If you followed the real-world block guide, you already should have a `main.js` in the `<plugin root>/src/Resources/app/administration/src` directory.
In this file, we need to import our new element's directory. So let's do it:
```